src/grasshopper/lib/util/utils.py

```diff
@@ -37,20 +32,13 @@ def wrapper(journey_object, *args, **kwargs):
             result = func(journey_object, *args, **kwargs)
             end_time = datetime.now()
             time_delta = end_time - start_time
-            tags.update(
-                {
-                    extra_tag_key: test_parameters.get(extra_tag_key)
-                    for extra_tag_key in extra_tag_keys
-                }
-            )
-            tags["environment"] = host
```

Comment:
is this being added in someplace else? i thought we for sure would want the host included in the tags?

Reply:
its being done here: https://github.com/alteryx/locust-grasshopper/blob/cdaf03f25058c54a7cc7f3f7cb69074a084ca3fe/src/grasshopper/lib/journeys/base_journey.py#L97

so, no need to re-do it here! Tested and looks good.